src/pages/receipts/csvFileReceipt.js

```diff
@@ -88,11 +113,38 @@ const modifyBSIQueryResults = (results) => {
   filteredResults = filteredResults.flat()
   filteredResults.forEach( i => {
       let vialMappings = getVialTypesMappings(i)
-       updateResultMappings(i, vialMappings)
+      updateResultMappings(i, vialMappings)
   })
   return filteredResults
 }
 
+const updateInTransitMapping = (results) => {
```

Comment:
It's not straightforward to know the data structure of variables `results`, `i` and `k`  in `updateInTransitMapping` function. I suggest using more meaningful names for these variables, for example `boxArray`, `box` and `bag`.

Reply:
done